content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -0,0 +1,245 @@
+created before the total number of hours required to solve the problem have been entered on a
+timesheet. In this case, a service product with the invoicing policy set to :guilabel:`Based on
+Timesheets` (like the one created above) would be used.
+
+Create a sales order with a time-tracked product
+------------------------------------------------
+
+To invoice a customer for post-paid support services, first create a sales order (SO) with the
+*support services product*. To do this, go to :menuselection:`Sales --> Orders --> Quotations -->
+New`.
+
+Fill out the quotation with the :guilabel:`Customer` information.
+
+On the :guilabel:`Order Lines` tab, click :guilabel:`Add a Product`. Select the post-paid services
+product configured in the steps above. After updating any other necessary information,
+:guilabel:`Confirm` the quotation.
+
+.. note::
+   Unlike with the prepaid services quotation, Odoo will not allow an invoice to be created at this
+   time. That is because no services have been performed. In other words nothing has been delivered.
+   Therefore, there is nothing to invoice.
+
+Create a helpdesk ticket for time-tracked services
+--------------------------------------------------
+
+To record a *Timesheet* entry for time-tracker services, go to :menuselection:`Helpdesk` and select
+the appropriate :guilabel:`Team`.
+
+If there is already an existing ticket for this issue, select it from the kanban view. This will
+open the ticket details form. If there is no existing ticket for this customer issue, click
+:guilabel:`New` to create a new ticket and enter the necessary customer information on the blank
+ticket details form.
+
+After selecting or creating a ticket, go to the :guilabel:`Sales Order Line` drop-down. Select the
+:abbr:`SO (sales order)` created in the previous step.
+
+Track support hours on a ticket
+-------------------------------
+
+In order to create an invoice for a product based on timesheets, hours need to be tracked and
+recorded. At this point, the service is considered *delivered*. To record hours for this support
+service, click on the :guilabel:`Timesheets` tab of the ticket.
+
+Click :guilabel:`Add a Line` to record a new entry. Select an :guilabel:`Employee` from the
+drop-down, and record the time spent in the :guilabel:`Hours Spent` column.
+
+Repeat these steps as needed until all time spent on the issues has been recorded.
+
+.. image:: track_and_bill/track-bill-record-timesheet-hours.png
+   :align: center
+   :alt: View of the timesheets tab on a helpdesk ticket.
+
+Create an invoice for hours tracked on a ticket
+-----------------------------------------------
+
+After the customer's issue has been solved, and it is determined no new timesheet entries will be
+made, an invoice can be created, and the customer can be billed.
+
+To do this, return to the :abbr:`SO (sales order)` by clicking on the :guilabel:`Sales Order` smart
+button at the top of the :guilabel:`Helpdesk Ticket`.
```

Comment:
Noting this as I could see the guilabel use either way here, differing to @StraubCreative's expertise 🙂
```suggestion
button at the top of the *Helpdesk* ticket.
```

Reply:
I will refer to the "After Sales" doc, since there were a lot of references to smart buttons in that one. In that instance, we settled on the following format- 'A :guilabel:`Return` smart button will then be added to the top of the
ticket.'
So guilabel for the smart button, and then just 'ticket' for the actual ticket (no Helpdesk). @StraubCreative, let me know if you disagree. 👍

Reply:
> I will refer to the "After Sales" doc, since there were a lot of references to smart buttons in that one. In that instance, we settled on the following format- 'A :guilabel:`Return` smart button will then be added to the top of the ticket.' So guilabel for the smart button, and then just 'ticket' for the actual ticket (no Helpdesk). @StraubCreative, let me know if you disagree. 👍

Agree with this.
It's the most straightforward option without forcing the need for unnecessary formatting